visualization/src/pcl_visualizer.cpp

```diff
@@ -4079,7 +4085,7 @@ pcl::visualization::PCLVisualizer::renderViewTesselatedSphere (
 
     poses.push_back (pose_view);
 
-  }
+}
 }
```

Comment:
Indentation here seemed correct. Having two `}` in consecutive lines with no indentation doesn't. What happened exactly?